doc/index.md

```diff
@@ -73,20 +75,46 @@ connectivity with PureCloud. Initialize must be called before any events will tr
 `sdk.createMedia(options) : Promise<MediaStream>` - Creates a media stream with audio, video, or both.
   - parameters
     - `Object options`: Required
-      - `Boolean video`: Optional; the returned stream will a video track.
-      - `Boolean audio`: Optional; the returned stream will have an audio track.
+      - `Boolean | String video`: Optional; the returned stream will a video track (`true` will use the sdk default device)\*.
+      - `Boolean | String audio`: Optional; the returned stream will have an audio track (`true` will use the sdk default device)\*.
+
+`sdk.getDisplayMedia() | Promise<MediaStream>` - Creates a media stream from the screen (this will prompt for user screen selection)
+
+`sdk.updateOutputDevice(deviceId) : Promise<void>` - Update the output device for _all active sessions_
+  - parameters
+    - `String deviceId`: Required; the desired output device ID\*
+
+`sdk.updateOutgoingMedia(updateOptions) : Promise<any>` - Update the outgoing media for a session
+  - paramters
+    - `Object updateOptions`: Required; with properties
+      - `String sessionId`: Optional; ID of the session to be updated (this or _session_ is required)
+      - `Session session`: Optional; session to be updated (this or _sessionId_ is required)
+      - `MediaStream stream`: Optional; media stream to update the session's outgoing media to (this supercedes deviceId(s))
+      - `String | Null videoDeviceId`: Optional; `string` for desired device ID, `null` for system default, `undefined` will not touch this media type on the session (superceded by `stream` option)\*
+      - `String | Null audioDeviceId`: Optional; `string` for desired device ID, `null` for system default, `undefined` will not touch this media type on the session (superceded by `stream` option)\*
+
+`sdk.updateDefaultDevices(options)` - Update the SDK's default device IDs
+  - parameters
+    - `Object options`: Optional
+      - `String | Null videoDeviceId`: Optional; Set the default video device ID. `string` for desired device ID, `null` for system default, `undefined` will not update the video device ID\*
+      - `String | Null audioDeviceId`: Optional; Set the default audio device ID. `string` for desired device ID, `null` for system default, `undefined` will not update the audio device ID\*
+      - `String outputDeviceId`: Optional; Set the default output device ID. `string` for desired device ID, `undefined` will not update the output device ID. NOTE: system default output devices are _not supported_. A valid `outputDeviceId` must be passed in to avoid errors\*
```

Comment:
System default output devices are not supported? Also, I think we may need a note here that states this depends on browser capabilities as well. Originally when we did this in web dir, only chrome supported this. I'm not sure if that has changed.